src/components/Button/Button.tsx

```diff
@@ -29,9 +37,10 @@ export interface ButtonProps {
  */
 const Button = ({
   primary = false,
```

Comment:
Should we just change primary to true by default, I mean reason nya dia dinamain primary karena dia default button